packages/esm-patient-vitals-app/src/vitals/paginated-vitals.scss

```diff
@@ -13,12 +19,22 @@
   }
 }
 
-.criticallyLow, .criticallyHigh {
+.criticallyHigh span, .criticallyLow span {
   border: 2px solid colors.$red-60 !important;
+  padding-left: 1rem;
+  padding-top: 0.25rem;
+  padding-bottom: 0.25rem;
 }
 
-.low, .high {
+.high span, .low span {
   border: 1.5px solid colors.$black-100 !important;
+  padding-left: 1rem;
+  padding-top: 0.25rem;
+  padding-bottom: 0.25rem;
```

Comment:
Why doesn't, e.g., `padding: 0.25rem` work? Also, it would be better to use Carbon's [spacing variables](https://carbondesignsystem.com/elements/spacing/overview/) instead of hard-coding various paddings...

Reply:
@ibacher the PR now has Carbon [spacing variables](https://carbondesignsystem.com/elements/spacing/overview/).
I din't use `padding: 0.25rem` which is now `$spacing-02` used `padding-left: $spacing-05` to align the abnormal values in line with the normal values which were not styled.

this is how the Datable looks with `padding: $spacing-02`

![0 25rem#](https://github.com/openmrs/openmrs-esm-patient-chart/assets/122885804/a8f8f362-a5ea-41a4-b5ec-b3410727ddb9)

Thanks!